package.json

```diff
@@ -28,14 +28,14 @@
     "q": "^1.0.0",
     "jsforce": "^1.1.2",
```

Comment:
Please update JSforce as well.

Reply:
The latest jsforce is 1.9.1 and since there is ^ it should pick the latest.